api/campaigns/class-maybe-show-campaign.php

```diff
@@ -124,6 +125,16 @@ function ( $post_data ) use ( $visit ) {
 				}
 			}
 
+			// Handle custom placements: Only one prompt should be shown per placement block.
+			// "Everyone" prompts should only be shown if the reader doesn't match any segments.
```

Comment:
I'm not finding that to be true in my testing. With two prompts in the custom placement, one assigned to non-subscribers, other to everyone, the latter is displayed (when viewing site as a non-subscriber). 

Seems that this happens simply because the everyone prompt is being evaluated earlier by this function, and so "takes the spot" of the prompt which should be shown in this custom placement. If the "everyone" prompts were evaluated last, this would work as expected, but I'm not sure if the prompts are ordered by segment priority.

Reply:
Ah, you're right. I think this needed the same kind of priority comparison we're doing for overlays and above headers, but per custom placement block instead of on the whole page. How does it work for you now? 02dff0f should show the single highest-priority prompt for each custom placement regardless of the order the prompts are evaluated in.